frontend/src/components/header/index.tsx

```diff
@@ -12,15 +12,28 @@ interface Props {
 
 const Header: React.FC<Props> = ({ children }) => {
   return (
-    <IonHeader>
-      <IonToolbar>
-        <IonButtons slot="start">
-          <IonBackButton />
-        </IonButtons>
-        <IonTitle>{children}</IonTitle>
-      </IonToolbar>
-    </IonHeader>
+    <>
+      <IonHeader>
+        <IonToolbar>
+          <IonButtons slot="start">
+            <IonBackButton />
+          </IonButtons>
+          <IonTitle>{children}</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+    </>
```

Comment:
Fragment can be removed as `<IonHeader>` can be the root.

```suggestion
<IonHeader>
  <IonToolbar>
    <IonButtons slot="start">
      <IonBackButton />
    </IonButtons>
    <IonTitle>{children}</IonTitle>
  </IonToolbar>
</IonHeader>
```